packages/components/CHANGELOG.md

```diff
@@ -4,12 +4,13 @@
 
 ### Bug Fix
 
+-   `Button`: Fix `iconSize` prop doesn't work with some icons ([#43821](https://github.com/WordPress/gutenberg/pull/43821)).
```

Comment:
We probably also want to mention that it was fixed in the `Icon` component.

Reply:
Fixed by 6c4d6a3